_posts/2015-01-31-getting-started.md

```diff
@@ -3,6 +3,19 @@ title:  "Getting Started"
 tags: chapters
 ---
 
+<h2 id="create-cycle-app">Using create-cycle-app</h2>
+
+The quickest way to create a new project with cycle, is by using [create-cycle-app](https://github.com/cyclejs-community/create-cycle-app), giving you the choice ES6 and TypeScript, using Browserify or Webpack as bundler tool.
+
+{% highlight text %}
+npm install --global create-cycle-app
+create-cycle-app my-awesome-app
+{% endhighlight %}
+
+This create a project called *my-awesome-app* (or the name you choosed) with Cycle *Run* and Cycle *DOM*.
```

Comment:
There are typos in this sentence.  I recommend rewriting to: "This will create a project called *my-awesome-app* (or a name you choose)  which uses the `xstream` stream library and includes Cycle's *Run* and *DOM* libraries.